testproject/testapp/tests/test_user_detail.py

```diff
@@ -27,11 +27,11 @@ def test_user_can_get_own_details(self):
 
         self.assert_status_equal(response, status.HTTP_200_OK)
 
-    def test_user_cannot_get_other_user_detail(self):
```

Comment:
The goal was to NOT allow common users see other users' profiles (at least with default config).